adding_numbers.py

```diff
@@ -0,0 +1,2 @@
+def sum(a,b):
+	return a*b
```

Comment:
I think it should be + instead of *.
```suggestion
	return a+b
```